tests/FakeItEasy.IntegrationTests/StringArgumentConstraints.cs

```diff
@@ -11,7 +11,11 @@ public static string StartsWith(this IArgumentConstraintManager<string> scope, s
 
         public static string Contains(this IArgumentConstraintManager<string> scope, string value)
         {
+#if FEATURE_STRING_CONTAINS_COMPARISONTYPE
+            return scope.Matches(x => x.Contains(value, StringComparison.CurrentCulture), $@"Contains ""{value}""");

```

Comment:
`String.Contains` now has multiple overloads. Not specifying the comparison type means current culture is used implicitly, which could be an issue and triggers CA1307. I specified current culture explicitly to keep the same behavior.